src/responder/common/cache_req/plugins/cache_req_group_by_id.c

```diff
@@ -64,6 +64,9 @@ cache_req_group_by_id_lookup(TALLOC_CTX *mem_ctx,
                              struct sss_domain_info *domain,
                              struct ldb_result **_result)
 {
+    if (cache_req_idminmax_check(data, domain)) {
```

Comment:
Here, please, introduce a errno_t ret var and do:

```
ret = cache_req_idminmax_check(data, domain);
if (ret != EOK) {
    return ret;
}
```

It's also easier to read and follows the coding style of the project,